packages/web-app-files/tests/unit/views/spaces/__snapshots__/Projects.spec.js.snap

```diff
@@ -1,39 +1,43 @@
-                      Rename
-                    </button></li>
-                  <li class="oc-spaces-context-action oc-py-xs oc-px-s"><button class="oc-button oc-button-m oc-button-justify-content-left oc-button-gap-m oc-button-passive oc-button-passive-raw"><span class="oc-icon oc-icon-m oc-icon-passive"><!----></span>
-                      Disable
-                    </button></li>
-                </ul>
-              </div>
-            </div> <a href="#/" aria-current="page" class="router-link-exact-active router-link-active"><span class="space-default-image oc-px-m oc-py-m oc-icon oc-icon-xxl oc-icon-passive"><!----></span></a>
-          </div> <span class="oc-card-body"><a href="#/" aria-current="page" class="oc-card-title router-link-exact-active router-link-active"></a> <p></p> <br> <!----></span>
+        <div class="spaces-list-card oc-card oc-card-default">
+          <div class="oc-card-media-top oc-border-b"><a href="#/" aria-current="page" class="router-link-exact-active router-link-active">
+              <!----> <span class="space-default-image oc-px-m oc-py-m oc-icon oc-icon-xxl oc-icon-passive"><!----></span>
+            </a></div> <span class="oc-card-body"><div class="oc-flex oc-flex-between oc-flex-middle"><div class="oc-flex oc-flex-middle"><span class="oc-mr-s oc-icon oc-icon-m oc-icon-passive"><!----></span> <a href="#/" aria-current="page" class="space-name oc-text-left router-link-exact-active router-link-active"><span></span></a>
+        </div>
+        <div><button aria-label="Show context menu" class="oc-button oc-button-m oc-button-justify-content-center oc-button-gap-m oc-button-passive oc-button-passive-raw" id="space-context-btn-1"><span class="oc-icon oc-icon-m oc-icon-passive"><!----></span></button>
+          <div id="space-context-drop-1" class="oc-drop oc-box-shadow-medium oc-rounded" options="[object Object]">
+            <div class="oc-card oc-card-body oc-rounded oc-background-highlight oc-p-s">
+              <ul class="oc-list oc-files-context-actions">
+                <li class="oc-spaces-context-action oc-py-xs oc-px-s"><button class="oc-button oc-button-m oc-button-justify-content-left oc-button-gap-m oc-button-passive oc-button-passive-raw"><span class="oc-icon oc-icon-m oc-icon-passive"><!----></span>
+                    Rename
+                  </button></li>
+                <li class="oc-spaces-context-action oc-py-xs oc-px-s"><button class="oc-button oc-button-m oc-button-justify-content-left oc-button-gap-m oc-button-passive oc-button-passive-raw"><span class="oc-icon oc-icon-m oc-icon-passive"><!----></span>
+                    Disable
+                  </button></li>
+              </ul>
+            </div>
+          </div>
         </div>
-      </li>
-    </ul>
   </div>
+  <p class="oc-text-left oc-mt-xs oc-mb-rm oc-text-truncate"><small></small></p></span>
+</div>
```

Comment:
Having those additional closing tags here gives me the worry that the respective template has some missing closing tags somewhere, could you look into it?

Reply:
don't see any tag fuck-up